lib/collection/src/shards/local_shard/telemetry.rs

```diff
@@ -59,8 +59,11 @@ impl LocalShard {
             vectors_size_bytes,
             payloads_size_bytes,
             num_points,
+            num_indexed_vectors,
         } = self.get_size_stats().await;
 
+        let num_unindexed_vectors = num_vectors.saturating_sub(num_indexed_vectors);
```

Comment:
Maybe it's better to explicitly count vectors on segments that don't have an index? 

I'm not sure if we want to count points in very small segments (below the indexing threshold) to be unindexed? If we don't, it'll be easier to count with the way I describe above.

Wdyt?